### PR TITLE
Add wrapper for HTTP response headers

### PR DESCRIPTION
**Pull request by jclin**
@jameswelle HTTP related wrappers work from https://github.com/articulate/Desktop/pull/6776

### CODE REVIEW

File: System.Doubles/Net/Http/HttpResponseMessageWrapper.cs [hunk 1]
```diff
@@ -28,6 +33,7 @@ public HttpResponseMessageWrapper(HttpResponseMessage httpResponseMessage)
 
             Content = httpResponseMessage.Content == null ? null : new HttpContentWrapper(httpResponseMessage.Content);
             RequestMessage = httpResponseMessage.RequestMessage == null ? null : new HttpRequestMessageWrapper(httpResponseMessage.RequestMessage);
+            ResponseHeaders = new HttpResponseHeadersWrapper(httpResponseMessage.Headers);
```
Comment (jameswelle) [inline System.Doubles/Net/Http/HttpResponseMessageWrapper.cs]: Do you know if it is possible for `httpResponseMessage.Headers` to be null here like the other properties above?

File: System.Doubles/Net/Http/HttpClientWrapper.cs [hunk 2]
```diff
@@ -5,11 +5,17 @@ namespace System.Net.Http
 {
     public sealed class HttpClientWrapper : IHttpClient
     {
+        public bool FollowRedirects
```
Comment (jameswelle) [inline System.Doubles/Net/Http/HttpClientWrapper.cs]: Can we keep this as a pure wrapper? This method lives on a handler in the implementation, right?

File: System.Doubles/Net/Http/HttpResponseMessageWrapper.cs (hunk 1 above)
Comment (jclin) [reply System.Doubles/Net/Http/HttpResponseMessageWrapper.cs]: The `Headers` are lazy loaded, so no worries in that regard.

File: System.Doubles/Net/Http/HttpClientWrapper.cs (hunk 2 above)
Comment (jclin) [reply System.Doubles/Net/Http/HttpClientWrapper.cs]: Sure. Updated.

**Comment by jameswelle** [comment]
looks good @jclin

**Pull request merged**